grakn-graql/src/main/java/ai/grakn/graql/internal/gremlin/fragment/AbstractFragment.java

```diff
@@ -30,14 +30,15 @@
     // TODO: Find a better way to represent these values (either abstractly, or better estimates)
     static final long NUM_INSTANCES_PER_TYPE = 100;
     static final long NUM_INSTANCES_PER_SCOPE = 100;
-    static final long NUM_RELATION_PER_CASTING = 10;
-    static final long NUM_SHORTCUT_EDGES_PER_INSTANCE = 10;
+    static final long NUM_RELATIONS_PER_CASTING = 10;
```

Comment:
Definitely needs to be a better method! @duckofyork is thinking about it I think.